Makefile

```diff
@@ -230,10 +232,17 @@ generate-crd-yaml:
 	@echo "Generating Contour CRD YAML documents..."
 	@./hack/generate-crd-yaml.sh
 
-.PHONY: generate-gateway-crd-yaml
-generate-gateway-crd-yaml:
+# TODO switch to version-specific YAML for the validating webhook
+# once v0.5.0 comes out (v0.4.x is using a certgen tool that's 
+# incompatible with Kubernetes 1.22+).
```

Comment:
I'll file an issue for this; we may get a v0.4.3 with the fix too before our next Contour release.

Reply:
Now using versioned webhook YAML.